### DISCUSSION TITLE
p/d6d8c1b89ef448f39a085e2ad5718cef/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#10] [Announcements]
# p/d6d8c1b89ef448f39a085e2ad5718cef/

好久不见的周报，希望 2023 新的一年，能有个新的开始。也能真正确定自己想做的各种事情，然后着手去做。 新的一年，吾辈在做什么呢？ 回家首先，由于国内的疫情躺平，今年春节终于能回老家了。不出所料的，春运的火车票一如既往的难买，虽然回去的票买到了，但却是凌晨 4 点的火车票。回来的票尚且在候补之中，实际什么时候能买到还没有着落。不过 8 月份的时候，在北京和老家都没有封城的时候，吾辈请了一周年假回

https://blog.rxliuli.com/p/d6d8c1b89ef448f39a085e2ad5718cef/

**Comment by b1mango** [comment]
eden好评🤗 新年快乐🎆🎉
